guide/the-user-interface.md

```diff
@@ -47,18 +47,18 @@ In order to reset the simulation, the following steps are performed:
   - The random seeds used by Webots internally are reset.
   - All the nodes are reset. This has the following implication depending on the node type:
     - **Brake**: The brake is released.
-    - **Charger**: The `battery` field and the `emissiveColor` field of the Material node of the first Shape child node are restored.
+    - **Charger**: The `battery` field and the `emissiveColor` field of the [Material](../reference/material.md) node of the first [Shape](../reference/shape.md) child node are restored.
```

Comment:
is not possible to add a link to the node written in bold too?

Reply:
I did not put them as link because I was afraid it will remove the bold, but maybe it is not the case, I will try.

Reply:
It seems to work fine:
![capture du 2018-06-21 16-41-23](https://user-images.githubusercontent.com/2461619/41726284-195bf2b2-7572-11e8-864a-fa2848cfe03a.png)
I will change all of them to links.